tests/queries/0_stateless/00398_url_functions.sql.j2

```diff
@@ -177,3 +177,12 @@ SELECT cutQueryStringAndFragment('http://paul@www.example.com/a/b/c?a=b#d=f');
 SELECT cutQueryStringAndFragment('//paul@www.example.com/a/b/c?a=b#d=f');
 SELECT cutQueryStringAndFragment('//paul@www.example.com/a/b/c#d=f');
 
+SELECT '====RESOLVE RELATIVE URL====';
+SELECT resolveRelativeURL('image.gif', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('image.gif', 'https://clickhouse.com/blog');
+SELECT resolveRelativeURL('/image.gif', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('//example.com/image.gif', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('ftp://example.com/image.gif', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('//', 'https://clickhouse.com/blog/');
+SELECT resolveRelativeURL('/image.gif', 'https://clickhouse.com/blog');
+SELECT resolveRelativeURL('/', 'https://clickhouse.com/blog/');
```

Comment:
Let's also add more corner-cases with empty and illegal strings, like containing only `/` or `:` in different combinations or event non-ascii characters

Reply:
And let's document in url-functions.md what happens in these situations (e.g. throws an exception, undefined behavior, returns an empty string, etc.)